cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSink.java

```diff
@@ -99,6 +99,7 @@
     private String csvSeparator;
```

Comment:
CNR entry missing ;)

Reply:
In addition, if this PR adds a configuration flag to disable metadata, I understand that flag should be described in documentation.

Reply:
Fixed in [08f8869](https://github.com/telefonicaid/fiware-cygnus/pull/1810/commits/08f886936d1432ae5b20e94cb223565672382d16)